bin/cron_script.sh

```diff
@@ -31,10 +31,13 @@ if [ "$HOST_CLOUD" = 'aws' ]; then
 
   # Check whether or not id string is empty
   if [ -n "$kubenow_latest_amiId" ]; then
-    # Updating source image id that will be used for AWS packer builder
+    # Updating source image id variable that will be used for AWS packer builder
     echo -e "Kubenow latest stable ami ID is: $kubenow_latest_amiId .\n"
     export AWS_SOURCE_IMAGE_ID="$kubenow_latest_amiId"
     echo -e "AWS Source Image ID is: $AWS_SOURCE_IMAGE_ID\n"
+    # Finally inserting new source image ID within the builder file
+    # shellcheck disable=SC2016
+    sed -i -e '/"source_ami"/c\         "source_ami":"{{user `source_image_id`}}",' build-"$HOST_CLOUD".json
```

Comment:
This was the reason of this glitch. While developing of the security updates cron script I forgot to include this line to make sure that the source image will match the latest stable image.